src/DGtal/geometry/doc/moduleAlphaThickSegmentReco.dox

```diff
@@ -0,0 +1,336 @@
+/* 
+ * Useful to avoid writing DGtal:: in front of every class.
+ */
+namespace DGtal {
+
+/**
+   
+@page moduleAlphaThickSegmentReco Alpha Thick Segment recognition   
+
+@writers Bertrand Kerautret 
+
+Part of the \ref packageGeometry.
+
+
+
+The Digital Straight Segment primitive (described here: @ref
+moduleArithDSSReco) is commonly used to extract geometric information
+from digital contours. It is exploited in various estimators as for
+instance in tangent \cite LachaudIVC2007 or curvature estimation 
+\cite KerautretLachaud2009 . However, this primitive is not efficient to
+directly process real contours with the potential presence of
+noise. To overcome this limitation the blurred segments was first
+introduced by Debled-Rennesson \i etal \cite debledFuzzy2005 which
+handle noise or contour irregularities.
+
+
+We present here the recognition of alpha thick segments as described
+in \cite debledFuzzy2005 and \cite FaureTangential2008. From a maximal
+thickness, it permits the recognition of a thick segment with the
+possibility to take into accounts some noise. Moreover the segment can
+be detected from points which are not necessary connected or with
+floating-point.
+
+@note The proposed implementation is mainly a backport from
+ [ImaGene](https://gforge.liris.cnrs.fr/projects/imagene) with some
+ various refactoring.
+
+
+[TOC]
+
+
+\section moduleAlphaThickSegmentRecoIntro  Alpha Thick Segment
+
+The first definition of \b blurred \b segment  \cite debledFuzzy2005 
```

Comment:
Blurred or alpha thick, please choose and keep it. In the intro, could add something saying that several names have been used to roughly the same geometrical objects..